### PR TITLE
docs: pl/container - add logs element to configuration xml file

### PR DESCRIPTION
**Pull request by mkiyama**
PR for 5X_STABLE
Will be ported to MAIN

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/extensions/pl_container.xml [hunk 1]
```diff
@@ -687,6 +687,17 @@ $$ LANGUAGE plcontainer;</codeblock></p>
...
                 </plentry>
+                <plentry>
+                  <pt>logs</pt>
+                  <pd>Optional. This element enables PL/Container logging for the container. Specify
+                    the attribute <codeph>enable="yes"</codeph>. The attribute
+                      <codeph>enable="no"</codeph> disables logging (the default).
+                    <codeblock>&lt;logs enable="yes">&lt;/logs></codeblock></pd>
+                  <pd>or<codeblock>&lt;logs enable="yes" /></codeblock><p>On Red Hat 7 or CentOS 7
+                      systems, the log is sent to the <codeph>journald</codeph> service. On Red Hat
+                      6 or CentOS 6 systems, the log is sent to <codeph>syslogd</codeph> service.
```
[the remark this answers is not in the document]
Comment (mkiyama) [reply gpdb-doc/dita/ref_guide/extensions/pl_container.xml]: fixed typo
@paul-guo- GPDB 5.3.0 supports SLES, will PL/Container be supported on SLES 11 and 12?

[the remark this answers is not in the document]
Comment (paul-guo-) [reply gpdb-doc/dita/ref_guide/extensions/pl_container.xml]: No.

**Comment by paul-guo-** [comment]
Looks good.

**Comment by mkiyama** [comment]
@dyozie This PR should be committed after https://github.com/greenplum-db/gpdb/pull/4197

4197 is for 5.3.0 and 5.2.0, this PR is for 5.4.0 (5.x-next)
sorry for the confusing, intertwined PRs

**Comment by mkiyama** [comment]
Closing PR - Updated pl/container for 5.x-next changes plcontainer command and XML schema.
PL/Container logging will be in the updates for those updates.

**Pull request closed without merging**